beta/kustomize/README.md

```diff
@@ -0,0 +1,196 @@
+# Deploy 1Password SCIM Bridge on Kubernetes with Kustomize
+
+*Learn how to deploy 1Password SCIM Bridge on a Kubernetes cluster using Kustomize.*
+
+> **Note**
+>
+> If you use Azure Kubernetes Service, learn how to [deploy the SCIM bridge there](https://support.1password.com/scim-deploy-azure/).
+
+**Table of contents:**
+
+- [Before you begin](#before-you-begin)
+- [Step 1: Deploy 1Password SCIM Bridge to the Kubernetes cluster](#step-2-deploy-1password-scim-bridge-to-the-kubernetes-cluster)
+- [Step 2: Create a DNS record](#step-3-create-a-dns-record)
+- [Step 3: Configure Let's Encrypt](#step-4-configure-lets-encrypt)
+- [Step 4: Test the SCIM bridge](#step-5-test-the-scim-bridge)
+- [Step 5: Connect your identity provider](#step-6-connect-your-identity-provider)
+- [Update your SCIM Bridge](#update-your-scim-bridge)
+- [Appendix: Resource recommendations](#appendix-resource-recommendations)
+- [Appendix: Customize your deployment](#appendix-customize-your-deployment)
+
+## Before you begin
+
+Before you begin, familiarize yourself with [PREPARATION.md](/PREPARATION.md) and complete the necessary steps there.
+
+### In this folder
+
+- [`/base`](./base/): The default deployment that the overlays build on top of to
+deploy your preferred bridge.
+- [`/overlays`](./overlays/): The overlays to customize your deployment.
+
+## Step 1: Prepare your `scimsession` file
+
+After downloading your `scimsession` file after creating your SCIM bridge credentials,
```

Comment:
```suggestion
After downloading your `scimsession` file generated when you set up the integration while signed into your 1Password account,
```

Minor tweak to avoid repeated "after" though I may have made it worse.